### DISCUSSION TITLE
Plotting slowed down after a restart

### DISCUSSION THREAD

**Discussion by fabriziocacicia** [swar/Swar-Chia-Plot-Manager#451] [Q&A]
Hi all. I'm struggling to find a solution to my problem.
My plotting speed was good the other day. Then I rebooted my system due to a connection problem and after that, all my attempts to plot at a decent speed had failed.

I have an i5-11400 with 32GB of DDR4 3000mhz and an SSD 1TB Gigabyte M.2 PCIe 3.0.

I have two screenshot to do a comparison between the two situations:

Before: 
<img width="897" alt="Screenshot 2021-05-17 at 00 32 32" src="https://user-images.githubusercontent.com/7525888/118414874-6e39af80-b6a7-11eb-97a0-c69e59029067.png">

After:
<img width="1131" alt="Screenshot 2021-05-17 at 00 25 16" src="https://user-images.githubusercontent.com/7525888/118414880-785bae00-b6a7-11eb-8327-5d855f8df508.png">

I took the two screenshots in about the same elapsed time, and as you can see the difference in the progress of the plots is very high.
I didn't change any settings in the config file and I can't figure out what can be the issue.

Would anyone be able to help me? Thanks

**Comment by Kali123411** [comment]
What is the temperature of your plotting drive? If the temp gets to hot the drive will start throttling and slow down your speed.

**Reply by fabriziocacicia** [reply]
Thanks for the fast reply.
This is the output of the command `sudo nvme smart-log /dev/nvme0`

<img width="426" alt="Screenshot 2021-05-17 at 00 43 51" src="https://user-images.githubusercontent.com/7525888/118415153-071cfa80-b6a9-11eb-8676-c409a39e71b3.png">
 
 Could the _Temperature Sensor 1_ be a problem?
 
 Is there a command on Linux that shows the throttling status of the SSD?

**Reply by fabriziocacicia** [reply]
I'm not sure if the actual temperature of the drive is the one above (_temperature_) or the one below (_Temperature Sensor 1_)

**Reply by Kali123411** [reply]
Normal operating temps are usually  0 - 70°C on nvme ssd. I try to keep mine well below 60.

**Reply by Kali123411** [reply]
You don't have anything set in the thermal management counts so I don't think temp is your problem.

**Reply by fabriziocacicia** [reply]
What I noticed is that the CPU usage dropped significantly. Before was almost always above 40% and often around 50%. Now it stays always near 10%, even with 3 concurrent plots. I tried both with 3 and 4 threads per plot (my CPU has 12 threads).

I also noticed that after noticing the slowdown the CPU started to spend most of its time at 800mhz and after some research, I figured out that the governor was set to _powersave_ and I had to manually set it to `performance`. Doing this I was able to run it again at full 4200mhz speed most of the time. But still, the performances are slow.
